app/backend/aquifers/migrations/0036_auto_20210202_1727.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.17 on 2021-02-02 17:27
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aquifers', '0035_aquifer_retire_date_20200522_2157'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='aquifer',
+            name='mapping_year',
+            field=models.PositiveIntegerField(blank=True, help_text='Use the following format: <YYYY>', null=True, validators=[django.core.validators.MinValueValidator(1990), django.core.validators.MaxValueValidator(2021)], verbose_name='Date of Mapping'),
```

Comment:
Would it be better to write a custom validator that ensures no future dates (in this case, just the current year) can be entered?
https://docs.djangoproject.com/en/3.1/ref/validators/

Reply:
I agree...   What do you think of tackling this bug in another issue/PR? I made https://apps.nrs.gov.bc.ca/int/jira/browse/WATER-1621 to track it.

The migration was auto-created by Django; I can leave it in to at least bump the max value from 2020 to 2021 or I can remove it from the PR.